test/test_framework.py

```diff
@@ -96,9 +96,9 @@ class MyEvent(EventBase):
             pass
 
         class MyNotifier(Object):
-            foo = Event(MyEvent)
-            bar = Event(MyEvent)
-            baz = Event(MyEvent)
+            foo = MyEvent
+            bar = MyEvent
+            baz = MyEvent
```

Comment:
I expect there will be a whole bunch of conflicts between these test changes and Dmitrii's changes, but I'm happy to sort out the conflicts, depending on what the merge order ends up being.